evaluate/evaluate.go

```diff
@@ -148,6 +150,7 @@ func Evaluate(ctx *Context) {
 								} else {
 									runCount = rl + 1
 								}
+								runID := ctx.RunIDOffset + runCount
```

Comment:
If RunIDOffset is by default `1` shouldn't all evaluations where it is not set automatically start with `runID: 2`

Reply:
Not sure what you mean here.

Reply:
Yeah an offset is usually something that is added. So I also would expect
- offset 0: first run ID 1
- offset 1: first run ID 2
- ...

Reply:
We call it `run-id-starts-at` in the CLI though which is no offset. So I would keep it.